bin/create-bioentities-suggesters.sh

```diff
@@ -12,6 +12,11 @@ COLLECTION=${SOLR_COLLECTION:-"bioentities-v${SCHEMA_VERSION}"}
 
 #############################################################################################
 
+printf "\n\nDelete search component for suggesters...\n"
+curl $SOLR_AUTH -X POST -H 'Content-type:application/json' --data-binary '{
+  "delete-searchcomponent": "suggest"
+}' http://${HOST}/solr/${COLLECTION}/config
```

Comment:
It's generally a good practice to quote variables to handle cases where they might contain spaces or special characters

You may want to add error handling in case the curl command fails.
```suggestion
}'  "http://${HOST}/solr/${COLLECTION}/config" || { echo "Error: Failed to delete search component"; exit 1; }
```

Reply:
Addressed in 9c698367950458b6218006ae673de0b01ed51a0b.